### ISSUE TITLE
Error: flatpickr: invalid locale fr

### ISSUE THREAD

**Issue by romainpoirier** [solspace/craft3-freeform#257]
I have a form with a _Date_ field, where the option `Use the Freeform datepicker for this field?` is not checked. This form is shown within a localized website: `<html lang="fr">`.

On form init, I get this error in the JS console: `Error: flatpickr: invalid locale fr`.

Following the [flatpickr doc](https://flatpickr.js.org/localization/), I have added this additional script:
`<script src="https://npmcdn.com/flatpickr/dist/l10n/fr.js"></script>`

But without any change, I tried also to add this after:
```
form.addEventListener('flatpickr-before-init', function(event) {
	flatpickr.localize(flatpickr.l10ns.fr);
});
```

Unfortunately without any change... This JS error is still shown, and the flatpickr not showing.

Any idea why and how to fix this?

**Craft & Plugin Info:**

- Craft Version: 3.7.25.1
- Freeform Version: 3.12.9.1
- Fresh Install or Upgrade: Upgrade

**Comment by kjmartens** [comment]
Hi @romainpoirier, sorry for the delay. I can duplicate this in my testing as well. We'll see what's up and work on a solution.

That said, it does appear that the usual approach of setting the Craft site locale to `fr` takes effect. Have you tried that approach, or is it something that won't work for your situation? :)

**Comment by kjmartens** [comment]
We were unable to get a template-level approach working at this time, but have added a new setting for Date & Time fields in the form builder that lets you set/force a locale for the field. Is that a solution that will work for your needs, @romainpoirier? 🙂

**Comment by romainpoirier** [comment]
> Hi @romainpoirier, sorry for the delay. I can duplicate this in my testing as well. We'll see what's up and work on a solution.

Thank you. I'm waiting for this fix to deploy my website.

> That said, it does appear that the usual approach of setting the Craft site locale to fr takes effect. Have you tried that approach, or is it something that won't work for your situation? :)

This is a single language website. The `fr` language is set at `/admin/settings/sites/1` using `Language: fr — French`.
It's not working, and I had this setup before submitting this ticket.

> but have added a new setting for Date & Time fields in the form builder that lets you set/force a locale for the field

Is it already added to the latest release (`v3.12.10`), or should I wait?

**Comment by kjmartens** [comment]
@romainpoirier It is available in the repo if you update to `dev-master`, but we'll likely release the update (3.12.11) today. 🙂

**Comment by kjmartens** [comment]
This is now available in v3.12.11. 🙂
